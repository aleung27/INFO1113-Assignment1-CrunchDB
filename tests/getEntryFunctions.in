SET r 4
GET r
PUSH r 10101
APPEND r 0
GET r
PICk r 1
PLUCK r 1
GET r
POP r
POP r
GET r
