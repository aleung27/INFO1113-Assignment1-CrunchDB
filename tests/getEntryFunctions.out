> ok

> [4]

> ok

> ok

> [10101 4 0]

> 10101

> 10101

> [4 0]

> 4

> 0

> []

